modules/signon/class_signon.php

```diff
@@ -80,10 +80,9 @@ public function GetLansurfer($username, $password)
             fclose($handle);
 
             // Log out
-            $handle = fopen("http://lansurfer.com/user.phtml?action=logout?LS_Session=$LS_Session", 'r', false, stream_context_create($opts));
+            fopen("http://lansurfer.com/user.phtml?action=logout?LS_Session=$LS_Session", 'r', false, stream_context_create($opts));
```

Comment:
See above. Would remove code related to lansufer completely.

Reply:
See https://github.com/lansuite/lansuite/issues/222